src/hotspot/share/runtime/stubRoutines.cpp

```diff
@@ -215,7 +215,10 @@ void StubRoutines::initialize1() {
   if (_code1 == NULL) {
     ResourceMark rm;
     TraceTime timer("StubRoutines generation 1", TRACETIME_LOG(Info, startuptime));
-    _code1 = BufferBlob::create("StubRoutines (1)", code_size1);
+    // Add extra space for large CodeEntryAlignment
+    int max_stubs = 10;
```

Comment:
Thanks for your update.

There are about 30 stubs (max number of stubs) to be generated in `generate_initial()` for x86_64.
Why not `max_stubs = 30` ?

Reply:
I instrumented the align() call in my private build to count how many used align(CodeEntryAlignment), and I counted only 7, then I rounded up to 10.  Maybe I should change the name to max_aligned_stubs to make it more clear?

Reply:
`max_aligned_stubs` is fine to me.
Please also update the copyright year.
Thanks.